testapps/test_pillowtop/tests/data/all-pillow-meta.json

```diff
@@ -169,7 +155,7 @@
     },
     "xform-pillow": {
         "change_feed_type": "KafkaChangeFeed",
-        "checkpoint_id": "xform-pillow-test_xforms_2016-07-07-test_report_xforms_20160824_1708-test_hqusers_2017-09-07",
+        "checkpoint_id": "xform-pillow-test_xforms_2016-07-07-report_xforms_20160824_1708-test_hqusers_2017-09-07",
```

Comment:
This still refers to the `report forms` index (see comment above about removing the reference)